src/LinAlg/hiopMatrixSparseTriplet.cpp

```diff
@@ -558,22 +574,19 @@ void hiopMatrixSymSparseTriplet::addToSymDenseMatrixUpperTriangle(int row_start,
     WM[i][j] += alpha*values_[it];
   }
 }
+
+/** 
+ * @brief block of W += alpha*(this)^T 
+ * @note W contains only the upper triangular entries
+ * 
+ * @warning This method should not be called directly.
+ * Use addUpperTriangleToSymDenseMatrixUpperTriangle instead.
+ */
 void hiopMatrixSymSparseTriplet::transAddToSymDenseMatrixUpperTriangle(int row_start, int col_start, 
```

Comment:
This function has never been called in the current examples.
There may be a bug in this code. It calls `addToSymDenseMatrixUpperTriangle`, which doesn't perform transportation to the matrix.

Reply:
Same as before, I would suggest to create a separate issue to discuss this. Only changes to HiOp API we made in this PR were the ones that were absolutely necessary for porting to GPU.

Reply:
yes, @nychiang please put that on the TODO list